meson.build

```diff
@@ -194,6 +194,10 @@ if not get_option('snpe-support').disabled()
 endif
 
 # tensorrt
+nvinfer_dep = dependency('', required: false)
+nvparsers_dep = dependency('', required: false)
+cuda_dep = dependency('', required: false)
+cudart_dep = dependency('', required: false)
```

Comment:
If you've decided to define empty dependency instead of `tensorrt_deps`, please change other build scripts.

For example, in `nnstreamer_example/meson.build` and `ext/nnstreamer/tensor_filter/meson.build`, `tensorrt_deps` --> `tensorrt_support_deps`